SeaBattle/Sea.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class Sea
+    {
+        private List<Deck> Decks { get; set; }
```

Comment:
Why does the Sea has Decks in it? What are they for?

Reply:
Created method where are this list using.